config/datastores.js

```diff
@@ -13,6 +13,17 @@
  * https://sailsjs.com/config/datastores
  */
 
+function generateDialectOptions()  {
+  if (process.env.AZURE_DB_PG_SSL !== 'true'){
+    return {};
+  }
+  return {
+    'ssl': {
+      ca: require('fs').readFileSync('./BaltimoreCyberTrustRoot.crt.pem')
```

Comment:
Can you add `rejectUnauthorized: true,` above the ca: line? I noticed there was a message about this missing when testing locally.